README.rst

```diff
@@ -30,11 +30,60 @@ To build and install this package:
 - ``./setup.py install``
 
 Usage
------
+=====
+
+Quick Start
+-----------
+
+In order to authenticate with the Linode API, you will first need to create a
+`Linode Personal Access Token`_ with your desired account permissions.
+
+The following code sample can help you quickly get started using this package.
+
+.. code-block:: python
+
+    from linode_api4 import LinodeClient, Instance
+
+    # Create a Linode API client
+    client = LinodeClient("MYPERSONALACCESSTOKEN")
+
+    # Create a new Linode
+    new_linode, root_pass = client.linode.instance_create(
+        "g6-nanode-1",
```

Comment:
nit: may worth also specifying the attributes' names here so a new user can better understand, i.e. 
```
type="g6-nanode-1",
region="us-southeast",
```